src/main/java/org/piwik/java/tracking/PiwikTracker.java

```diff
@@ -62,7 +55,7 @@ public PiwikTracker(String hostUrl) {
    * <strong>http://your-piwik-domain.tld/piwik.php</strong>.
    * @param timeout the timeout of the sent request in milliseconds
    */
-  public PiwikTracker(String hostUrl, int timeout) {
+  public PiwikTracker(final String hostUrl, final int timeout) {
```

Comment:
This "code changes" had been done by my IDE because of our company internal code convenients.
I can revert them if you wanted me to do ...

Reply:
They're fine, we use the same in our company as well.